driver-async/src/test/functional/com/mongodb/async/client/TransactionsTest.java

```diff
@@ -534,6 +531,9 @@ private ClientSession nonNullClientSession(@Nullable final ClientSession clientS
         List<Object[]> data = new ArrayList<Object[]>();
         for (File file : JsonPoweredTestHelper.getTestFiles("/transactions")) {
             BsonDocument testDocument = JsonPoweredTestHelper.getTestDocument(file);
+            if (!JsonTestServerVersionChecker.canRunTests(testDocument)) {
```

Comment:
If we make the check here, the test runner isn't aware that the test is being skipped, so it won't show up in the list of ignored tests.  Was that your intention?  I generally like being able to see the skipped tests show up as grey when running in IntelliJ, at the least.

Reply:
Yes - because we'd been using that pattern throughout the test suite

Reply:
I'll update and mark them as skipped instead of ignored

Reply:
Done